src/index.css

```diff
@@ -1,5 +1,252 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
+
+body {
+  margin: 20px;
+}
+
+.navigation-bar {
+  display: flex;
```

Comment:
- [ ] Still on the concerns of the previous review. Not a new issue.
  
  Let's strive to keep the code DRY. This property:
  ```CSS
  display: flex;
  ```
  has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.
  
  ### Example code
  ```CSS
  .d-flex {
    display: flex;
    align-items: center;
  }
  ```
  
  Then apply this class to HTML element that needs this behavior:
  
  ```HTML
  <section class="d-flex">
  ```
  
  OR, in JSX, if creating elements dynamically:
  ```JSX
  <section className="d-flex">
  ```
  
  The goal of DRY is that we write this rule only once in the CSS. Please refactored accordingly just as was required in the first review.